src/igvfd/types/file_set.py

```diff
@@ -542,34 +542,50 @@ def summary(self, request, file_set_type, scope, selection_criteria, genes=None,
         criteria = criteria + selection_criteria
 
         if scope == 'loci':
-            if len(loci) > 1:
+            if loci and len(loci) > 1:
```

Comment:
i think you need to include loci in the function parameters with loci=None (like genes)

Reply:
loci is already included in the parameters. 

    def summary(self, request, file_set_type, scope, selection_criteria, genes=None, small_scale_gene_list=None, large_scale_gene_list=None, guide_type=None,
                **loci=None**, exon=None, tile=None, associated_phenotypes=None):

I will fix this summary to remove the properties in a follow up ticket for the removal of the genes and loci properties.

Reply:
sorry, my bad, somehow I didn't notice loci in the parameters list